spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
[Ordinary objects have many internal slots](http://tc39.github.io/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots), so I think @anba is correct. There needs to be some verbage here about what kind of object the thing actually is. I think @rwaldron's suggestion is best - apply to all types below and then I can pull this in :)